project/threads/api.py

```diff
@@ -206,44 +207,44 @@ def create_civi(request):
             parent_civi = Civi.objects.get(id=related_civi)
             parent_civi.responses.add(civi)
 
-            if parent_civi.author.user.username != request.user.username:
+            if parent_civi.author.username != request.user.username:
                 notify.send(
                     request.user,  # Actor User
-                    recipient=parent_civi.author.user,  # Target User
+                    recipient=parent_civi.author,  # Target User
                     verb=u"responded to your civi",  # Verb
                     action_object=civi,  # Action Object
                     target=civi.thread,  # Target Object
                     popup_string="{user} responded to your civi in {thread}".format(
-                        user=a.full_name, thread=civi.thread.title
+                        user=user.full_name, thread=civi.thread.title
                     ),
                     link="/{}/{}".format("thread", thread_id),
                 )
 
         else:  # not a reply, a regular civi
             c_qs = Civi.objects.filter(thread_id=thread_id)
-            accounts = Profile.objects.filter(
+            users = get_user_model().objects.filter(
                 pk__in=c_qs.values("author").distinct()
             )
             data = {
                 "command": "add",
-                "data": json.dumps(civi.dict_with_score(a.id)),
+                "data": json.dumps(civi.dict_with_score(user.id)),
             }
 
-            for act in accounts:
-                if act.user.username != request.user.username:
+            for u in users:
+                if u.username != request.user.username:
                     notify.send(
                         request.user,  # Actor User
-                        recipient=act.user,  # Target User
+                        recipient=u.user,  # Target User
                         verb=u"created a new civi",  # Verb
                         action_object=civi,  # Action Object
                         target=civi.thread,  # Target Object
                         popup_string="{user} created a new civi in the thread {thread}".format(
-                            user=a.full_name, thread=civi.thread.title
+                            user=user.full_name, thread=civi.thread.title
                         ),
                         link="/{}/{}".format("thread", thread_id),
                     )
 
-        return JsonResponse({"data": civi.dict_with_score(a.id)})
+        return JsonResponse({"data": civi.dict_with_score(user.id)})

```

Comment:
It would be useful here to be explicit whether this is the `request.user`, since this code is somewhat long and convoluted.

```suggestion
        return JsonResponse({"data": civi.dict_with_score(request.user.id)})

```